### PR TITLE
Announce new seL4 release

### PR DESCRIPTION
**Pull request by axel-h**
Address https://github.com/seL4/website/issues/94

### CODE REVIEW

File: content/news.pml [hunk 1]
```diff
@@ -39,6 +39,49 @@
...
+<div>
+ <p>
+    <img src="images/seL4.svg" style="width:30%; padding-left:1em; float:right" alt="seL4">
+
+    The seL4 Foundation has released the following updates:
+    <ul>
+      <li>
+        seL4 v12.1.0:
+        <a href="https://github.com/seL4/seL4/releases/tag/12.1.0">release</a>,
+        <a href="https://github.com/seL4/seL4/blob/12.1.0/CHANGES">change log</a>
```
Comment (lsf37) [inline content/news.pml:54]: The link should go to the more nicely formatted change log on the docs site, I think, i.e.

```suggestion
        <a href="https://docs.sel4.systems/releases/sel4/12.1.0">change log</a>
```

File: content/news.pml [hunk 2]
```diff
@@ -39,6 +39,49 @@
...
+      </li>
+      <li>
+        CAmkES v3.10.0:
+        <a href="https://github.com/seL4/camkes-tool/releases/tag/camkes-3.10.0">release</a>,
+        <a href="https://github.com/seL4/camkes-tool/blob/camkes-3.10.0/CHANGES">change log</a>
```
Comment (lsf37) [inline content/news.pml:59]:
```suggestion
        <a href="https://docs.sel4.systems/releases/camkes/camkes-3.10.0">change log</a>
```

File: content/news.pml [hunk 3]
```diff
@@ -39,6 +39,49 @@
...
+      </li>
+      <li>
+        CapDL v0.2.1:
+        <a href="https://github.com/seL4/capdl/releases/tag/0.2.1">release</a>,
+        <a href="https://github.com/seL4/capdl/blob/0.2.1/CHANGES">change log</a>
```
Comment (lsf37) [inline content/news.pml:64]:
```suggestion
        <a href="https://docs.sel4.systems/releases/capdl/0.2.1">change log</a>
```

**Review by lsf37 — APPROVED** [review]
Good point, should definitely do that.

@nomadeel can you please merge this simultaneously with the mailing list announcement when everything is fully finalised?

**Comment by lsf37** [comment]
(The deploy check only works from in-repo branches, on the list to be fixed, but somewhat nontrivial. The layout it good, I checked locally).

**Comment by axel-h** [comment]
links changed.

**Comment by gernotheiser** [comment]
thanks for this @axel-h

**Comment by nomadeel** [comment]
Thanks @axel-h , you beat me to it.

**Pull request merged**